apps/shared/y4m.c

```diff
@@ -457,6 +457,7 @@ avifBool y4mWrite(const char * outputFilename, const avifImage * avif)
                     y4mHeaderFormat = "Cmono XYSCSS=400";
                     break;
                 case AVIF_PIXEL_FORMAT_NONE:
+                case AVIF_PIXEL_FORMAT_COUNT:
```

Comment:
Nit: The three comments at lines 461, 481, 501 should be updated to use plural nouns:

```
                    // will error later; these cases are here for warning's sake
```